Libraries/ToTripleSlashPorter.cs

```diff
@@ -245,9 +244,8 @@ private void FindSymbolInReferencedProjects(DocsType docsType, IEnumerable<Proje
                     Log.Info(false, $"Symbol for '{docsType.FullName}' not found in referenced project '{projectPath}'.");
                     if (n < projectReferences.Count())
                     {
-                        Log.Info(false, $" Trying the next project...");
+                        Log.Info(true, $" Trying the next project...");
```

Comment:
Same question about the newlines

Reply:
Same as above, the makes up for the writeline I removed.